tests/src/test/resources/kg/search/synapse.json

```diff
@@ -0,0 +1,93 @@
+{
+  "@context": "https://bbp.neuroshapes.org",
+  "@id": "https://bbp.epfl.ch/data/synapse",
+  "@type": [
+    "Entity",
+    "Parameter",
+    "SchafferCollateralAnatomyParameter",
+    "SynapsesPerConnection",
+    "ExperimentalSynapsesPerConnection"
+  ],
+  "brainLocation": {
```

Comment:
Minor but there is no real use of having most of these fields here, `synapticPathway` should be enough

Reply:
Can do, I guess you are right it isn't needed